### DISCUSSION TITLE
Slack reports

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#291] [Anything Else]
"It would be pretty awesome if you could set up daily/weekly/monthly reports to pushed over to a Slack channel. 

Email is nice, but Slack has turned into a real time business data feed. It would be great to get analytics piped in there as well, formatted nicely."

(feedback via email)

**Comment by ukutaht** [comment]
I don't have time to test this right now but it might be possible to just send email to a special slack address: https://slack.com/intl/en-ee/slack-tips/send-email-to-slack

**Reply by ukutaht** [reply]
Better link: https://slack.com/intl/en-gb/help/articles/206819278-Send-emails-to-Slack#set-up-a-forwarding-email-address

**Comment by metmarkosaric** [comment]
Here's how to send Plausible weekly/monthly reports and traffic spike notifications directly into a Slack channel: https://docs.plausible.io/slack-reports

**Reply by bryevdv** [reply]
@metmarkosaric Is there any intention to support a more sophisticated Slack integration? The current option of just dumping emails into a channel is fairly clunky. The messages are collapsed and have to be interacted with in order to view. If there are multiple sites then this an just annoyance multiplies. I just turned it off for that reason. Ideally, there multiple sites on a single account could be consolidated, formatted nicely, expanded and viewable at a glance, with links back to relevant dashboard pages for any drill-down, etc. Things like inline graphs would be super nice, if feasible.
